src/ItemRepository.js

```diff
@@ -0,0 +1,65 @@
+export default class ItemRepository {
+  constructor() {
+    this.todoItems = [];
+    this.loadItems();
+  }
+
```

Comment:
- Kindly add the [right linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) into your project.